tests/debugging/test_uploader.py

```diff
@@ -44,16 +44,13 @@ def test_uploader_full_buffer():
     size = 1 << 8
     with ActiveBatchJsonEncoder(size=size, interval=0.5) as uploader:
         item = "hello" * 10
-        n = size // len(item)
+        item_len = len(item)
+        n = (size + item_len - 1) // item_len
```

Comment:
I would keep this as `n = size // len(item)` and then do a loop `for _ in range(2 * n)` so that we are sure enough, but without doing an infinite loop. WDYT?

Reply:
reverted